code/lib/cli/src/initiate.ts

```diff
@@ -319,9 +320,53 @@ async function doInitiate(options: CommandOptions, pkg: PackageJson): Promise<vo
 
   logger.log('\nFor more information visit:', chalk.cyan('https://storybook.js.org'));
 
-  if (projectType === ProjectType.ANGULAR) {
-    logger.log('\nTo run your Storybook, type:\n');
-    codeLog([`ng run ${installResult.projectName}:storybook`]);
+  const shouldRunDev =
+    projectType !== ProjectType.REACT_NATIVE &&
+    process.env.CI !== 'true' &&
+    process.env.IN_STORYBOOK_SANDBOX !== 'true';
+  if (shouldRunDev) {
+    logger.log('\nRunning Storybook');
```

Comment:
Running Storybook automatically. To run it manually, use the following command: `xyz`

Reply:
@shilman @jonniebigodes here's how it looks like after the change:

For Vite:
![image](https://github.com/storybookjs/storybook/assets/1671563/3fc8fa83-2f15-42b3-85f2-f04c1dbdf4f8)


For Angular Webpack (the most verbose we have):
![image](https://github.com/storybookjs/storybook/assets/1671563/26946803-4e67-4f5d-b918-39ee98878ff8)


Thoughts?

Reply:
Looks great! I'm sure we can clean all this up if we rework the CLI, but for now I'd say ship it!